Src/Support/Google.Apis.Auth.Tests/JsonWebSignatureTests.cs

```diff
@@ -99,12 +108,37 @@ public async Task InvalidRS256Signature()
             Assert.Equal("JWT invalid, unable to verify signature.", ex.Message);
         }
 
+#if NETCOREAPP2_0 || net452 || net46
+        [Fact]
+        public async Task InvalidES256Signature()
+        {
+            // Lets just change the last character of the signed token to break the signature.
+            // But Headers and Payload are still valid so we guarantee that the only
+            // failure comes from the signature being invalid.
+            // The last character was an A, changing for a 4. If at some point the token used in tests changes
+            // and the new token ends in anything but a 4, this test will still be valid. If the new token ends in a 4
+            // this test will fail and then we'll have to replace the 4 by an A or any other character.
+            var invalidToken = FakeCertificateCache.Es256ForIap.Substring(0, FakeCertificateCache.Es256ForIap.Length - 1) + "4";
+            var ex = await Assert.ThrowsAsync<InvalidJwtException>(() =>
+                JsonWebSignature.VerifySignedTokenAsync(invalidToken, BuildOptions(new MockClock { UtcNow = FakeCertificateCache.ValidEs256ForIap })));
+            Assert.Equal("JWT invalid, unable to verify signature.", ex.Message);
+        }
+#else
```

Comment:
As above - maybe even extract this out to a block earlier on, so it defines a single symbol of ES256_SUPPORTED (for .NET framework and netcore2.x), or doesn't (for netcore1.x) or errors (if we haven't specified)